packages/ethers5/exports/react.tsx

```diff
@@ -46,11 +48,26 @@ export function useWeb3ModalAccount() {
   const address = state.address
   const isConnected = state.isConnected
   const chainId = state.chainId
+  const provider = state.provider
+  const providerType = state.providerType
+
+  async function disconnect() {
```

Comment:
Let's keep consistent and do `useDisconnect` that returns disconnect function. Also, it seems like same logic as in `client` is replicated there. Can we not just use `ConnectionController.disconnect` at this stage (same as all other hooks use controllers for things like theme etc?)